modules/react/text/stories/BodyText.stories.mdx

```diff
@@ -33,4 +31,4 @@ You may override the rendered element using the `as` prop.
 
 ## Props
 
```

Comment:
This isn't needed anymore and appears like this:

<img width="718" alt="Screenshot 2023-02-23 at 1 22 35 AM" src="https://user-images.githubusercontent.com/1075861/220855447-4f1454ee-fd0a-4a07-8c7a-8b8c104cc62f.png">


```suggestion
```